apps/application-system/api/src/app/modules/application/files/utils/pdf.ts

```diff
@@ -11,13 +14,28 @@ import {
 import { PdfConstants } from './constants'
 import { DistrictCommissionerLogo } from './districtCommissionerLogo'
 
+const formatDays = (date: string): string => {
```

Comment:
At first I wasn't going to do this but decided to humor the content creators. Instead of displaying 
"Fyrirspurn og uppfletting í gögnum Þjóðskrár fór fram mánudagur 25. janúar 2021 kl. 13:36."
Then we have 
"Fyrirspurn og uppfletting í gögnum Þjóðskrár fór fram mánudaginn 25. janúar 2021 kl. 13:36."